static/styles/drafts.css

```diff
@@ -0,0 +1,166 @@
+.drafts {
+    margin-top: 55px;
+    padding-left: 15px;
+    -webkit-font-smoothing: antialiased;
+}
+
+.drafts-container {
+    position: relative;
+    height: 95%;
+    background-color: #fff;
+    border-radius: 4px;
+    padding: 0px;
+    width: 60%;
+    overflow: hidden;
+    max-width: 1200px;
+    max-height: 1000px;
```

Comment:
Maybe this max-width should be set so that the drafts area is never bigger than the max width of the Zulip message feed?   Visually, I think that's a bit smaller than this.